packages/expo-file-system/src/FileSystem.types.ts

```diff
@@ -1,16 +1,41 @@
+export enum FileSystemSessionType {
+  BACKGROUND = 0,
+  FOREGROUND = 1,
+}
+
 export type DownloadOptions = {
   md5?: boolean;
   cache?: boolean;
   headers?: { [name: string]: string };
+  sessionType?: FileSystemSessionType; // iOS only
```

Comment:
```suggestion
  /**
   * iOS only
   */
  sessionType?: FileSystemSessionType;
```
Let's make it stands out 😊